app/modules/map/lib/config.js

```diff
@@ -8,7 +8,7 @@ export default {
     attribution: `Map data &copy; <a href='http://openstreetmap.org'>OpenStreetMap</a> contributors,
 <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>,
 Imagery © <a href="http://mapbox.com">Mapbox</a>`,
-    minZoom: 2,
+    minZoom: 6,
```

Comment:
This is problematic if items are displayed on a global scale. Lets say main user position is in indonesia, and there is an item in peru, map bounds will then be exactly in between those two marker locations (main user marker and user item marker), with a 6 zoom, so basically a map that show nothing.
I see two workarounds: 
- either the map does not display markers further than x km from main user (but that too bad since inventaire have this information) 
- or minZoom is 2 as default and a component variable could adjust it if necessary.

Reply:
My primary motivation was that when then map is too zoomed-out, only partial results might be displayed anyway, but I didn't think of the case you're referring to. Reverted in fa8287286

Reply:
My primary motivation was that when then map is too zoomed-out, only partial results might be displayed anyway, but I didn't think of the case you're referring to. Reverted in e1565cdbb